packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -136,17 +142,73 @@ describe('HCP home page', () => {
       render(getHomePage())
     })
 
-    const patientRow = screen.queryByTestId(`patient-row-${unMonitoredPatient.userId}`)
-    const removeButton = within(patientRow).getByRole('button', { name: `Remove patient-${unMonitoredPatient.email}` })
+    const patientRow = screen.queryByTestId(`patient-row-${unmonitoredPatient.userId}`)
+    const removeButton = within(patientRow).getByRole('button', { name: `Remove patient-${unmonitoredPatient.email}` })
     removeButton.click()
     const removeDialog = screen.getByRole('dialog')
     const confirmRemoveButton = within(removeDialog).getByRole('button', { name: 'Remove patient' })
 
     await act(async () => {
       confirmRemoveButton.click()
     })
-    expect(removePatientMock).toHaveBeenCalledWith(unMonitoredPatient.teamId, unMonitoredPatient.userId)
+    expect(removePatientMock).toHaveBeenCalledWith(unmonitoredPatient.teamId, unmonitoredPatient.userId)
     expect(screen.getByTestId('remove-hcp-patient-dialog')).toBeInTheDocument()
     expect(screen.getByTestId('alert-snackbar')).toHaveTextContent('Impossible to remove patient. Please try again later.')
   })
+
+  it('should display Add patient dialog with appropriate error messages depending on the input user and the selected team', async () => {
+    await act(async () => {
+      render(getHomePage())
+    })
+
+    const secondaryBar = screen.getByTestId('patients-secondary-bar')
+    const addPatientButton = within(secondaryBar).getByRole('button', { name: 'Add patient' })
+    expect(addPatientButton).toBeInTheDocument()
+
+    addPatientButton.click()
```

Comment:
```suggestion
    userEvent.click(addPatientButton)
```
userEvent import:
`import userEvent from '@testing-library/user-event'`
This allows to mimic the user behaviour in a better way.
https://github.com/testing-library/user-event#the-problem